pkg/cli/admin/nodeimage/create.go

```diff
@@ -71,6 +86,13 @@ var (
 
 		# Specify a custom image name
 		  oc adm node-image create --o=my-node.iso
+
+		# Create an ISO to add a single node without using the configuration file
+		  oc adm node-image create --mac-address=00:d8:e7:c7:4b:bb
```

Comment:
What effect does this actually have?

Reply:
@andfasano indicated that --mac-address is supposed to be the only flag that is required if the nodes-config.yaml file is not specified. But after testing this patch, I can enter any valid mac-address and the node still is added to the cluster if the node has an IP address configured on the DHCP server.